Parse/ParseClientConfiguration.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2015-present, Parse, LLC.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
```

Comment:
Maybe add import for Foundation? Since we rely on `NSObject`/`NSCopying`/`NSString` here?